Pod/Classes/MRGPagerController.h

```diff
@@ -48,12 +48,17 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (void)setCurrentViewController:(UIViewController *)currentViewController animated:(BOOL)animated;
 
+- (void)updateBadges;
+
 @end
 
 @protocol MRGPagerControllerDelegate<NSObject>
 
 - (void)pagerController:(MRGPagerController *)pagerController didEndScrollingOnViewController:(UIViewController *)viewController;
```

Comment:
Meme si c'est le default, ca fait plus clair quand y a un optional aussi:

```suggestion
@required
- (void)pagerController:(MRGPagerController *)pagerController didEndScrollingOnViewController:(UIViewController *)viewController;
```